data/json/items/gun/38.json

```diff
@@ -90,8 +90,15 @@
     "durability": 8,
     "blackpowder_tolerance": 56,
     "clip_size": 6,
-    "magazines": [ [ "38", [ "38_speedloader6" ] ] ],
-    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "38": 6 } } ]
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "holster": true,
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "item_restriction": [ "38_speedloader6" ]
```

Comment:
so all of the guns with integral magazines that can be used with speedloaders should still have an integral magazine, not a magazine well - in the reloading code it just allows you to use a speedloader to put the ammo into the gun, so it should be a MAGAZINE type